package.json

```diff
@@ -66,13 +66,15 @@
     "copy-webpack-plugin": "^6.4.1",
     "crlf": "^1.1.1",
     "css-loader": "^5.2.4",
-    "eslint": "^7.29.0",
-    "eslint-config-airbnb": "^18.2.1",
-    "eslint-plugin-import": "^2.23.4",
+    "eslint": "^7.2.0",
```

Comment:
Is the downgrade on purpose? I think it doesn't have any effect since it uses 7.30.0 anyway according to the lock file.